axolotl-web/src/store/store.js

```diff
@@ -174,6 +177,22 @@ export default new Vuex.Store({
       })
 
     },
+    SET_MESSAGE_UPDATE(state, message) {
+      if (state.messageList.ID == message.ChatID) {
+        var index = state.messageList.Messages.findIndex(m => {
+          if (m.ID === message.ID) return true
```

Comment:
Could be shortened, and the comparison be returned straight away?

`return m.ID === message.ID`

Reply:
I could be, but that's a coding style i am not used to. We can change that.

Reply:
I dont want to push my opinions, it was more an attempt to introduce less code. Possibly a few lines in a CONTRIBUTING.md could make sure we are all aligned?

Reply:
Let's do that change here indeed.